analyzer.py

```diff
@@ -33,53 +33,30 @@ def split_subtests(self):
 
         return split_data
 
-    def ttest(self):
-        """
-        Take the data and perform a cross-ttest on the rows.
-        Data returned looks like the following:
-        ```
-            [
-                {
-                    'ttest': 7.2,
-                    'pval': 0.01,
-                    'name1': macosx-raptor,
-                    'name2': macosx-browsertime
-                }, ...
-            ]
-        ```
-
-        :return dict: List of results.
-        """
-        results = []
+    def get_header(self):
+        template_header_path = "testing/resources/template/header"
+        with open(template_header_path, "r") as f:
+            template_header_content = f.read()
+            return template_header_content
 
-        split_data = self.split_subtests()
+    def get_analysis_template(self,func):
+        template_function_folder_path = "testing/resources/analysis-templates/"
```

Comment:
This is a constant value so it should be placed at the top of the file within a variable like `NOTEBOOK_SECTIONS_PATH`